docker-assets/wait-for-container-env.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Sleep until /etc/container-environment is created by postStart hook
+Before=httpd
+[Service]
+Type=oneshot
+ExecStart=/bin/bash -c "until [ -f /etc/container-environment ]; do echo waiting for /etc/container-environment ; sleep 1; done"
```

Comment:
Shouldn't the `waiting for /etc/container-environment` be quoted?

Reply:
why?

Reply:
Readability?

Reply:
Do we even need an echo call there? I know it's nice to see some output but systemd will provide output if something is unusual with the execprestart command.